src/Utils/helpers.ts

```diff
@@ -40,3 +40,15 @@ export const MODE = {
     DEV: 'dev',
     EMULATOR: 'emulate',
 };
+
+export const validateName = (value: string) => {
+    const regex = /\d/;
+    if (value && value.includes('.')) return 'Name must not contain a dot';
+    if (value && Array.isArray(value))
+        for (let i = 0; i < value.length; i++) {
+            if (regex.test(value[i])) {
+                return 'Value must not contain a number';
+            }
+        }
+    if (value && regex.test(value)) return 'Value must not contain a number';
```

Comment:
move this to top. and bring that forloop one down. and also wrap the for loop one with { } brackets